.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     matrix:
       linux:
         CONFIG: linux64
-        IMAGE_NAME: condaforge/linux-anvil-comp7
+        IMAGE_NAME: condaforge/linux-anvil-cuda:9.2
```

Comment:
As this hack is still here I've added WIP to to title. You should be able to edit and remove it when this is reverted and the PR is to ready merge.